targetFunctions/common/loopMatlabCustomLayers/sourceCompile/loopMatlabCustomLayersWrapperXYPoints.m

```diff
@@ -1,4 +1,4 @@
-function [allLayers, allRoughs] = loopMatalbCustlayWrapper_XYContrasts(cBacks,cShifts,cScales,cNbas,cNbss,cRes,backs,...
+function [allLayers, allRoughs] = loopMatlabCustomLayersWrapperXYPoints(cBacks,cShifts,cScales,cNbas,cNbss,cRes,backs,...
      shifts,sf,nba,nbs,res,cCustFiles,numberOfContrasts,customFiles,params)
```

Comment:
`loopMatalbCustlayWrapper_XYContrasts` -> `loopMatlabCustomLayersWrapperXYPoints`?

Reply:
So, the function name now matches the filename. Why didn't it before?

Reply:
They do match on the master branch - I don't know where the contrasts comes from.